src/Components/Button/index.jsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styles from './styles.module.scss';
+
+export const Button = ({ clickHandler, buttonText, isDisabled }) => (
```

Comment:
I think we should stick to `handleClick, handleBlur, handleWhatever` naming for event handler functions